trigger/task.go

```diff
@@ -665,6 +666,31 @@ func shouldSendTaskRegression(sub *event.Subscription, t *task.Task, previousTas
 	return false, nil
 }
 
+func (t *taskTriggers) taskSuccessfulExceedsDuration(sub *event.Subscription) (*notification.Notification, error) {
+	if t.task.IsPartOfDisplay() {
+		return nil, nil
+	}
+
+	if t.task.Status != evergreen.TaskSucceeded {
+		return nil, nil
+	}
+
+	thresholdString, ok := sub.TriggerData[event.TaskDurationKey]
```

Comment:
Since this logic is duplicated from below, we can just call taskExceedsDuration after checking the status

Reply:
🤦🏻 yes, fixed